service/src/main/java/bio/terra/policy/db/PaoDao.java

```diff
@@ -371,11 +371,11 @@ private DbPao getDbPao(UUID objectId) {
     MapSqlParameterSource params =
         new MapSqlParameterSource().addValue("object_id", objectId.toString());
 
-    DbPao dbPao = tpsJdbcTemplate.queryForObject(sql, params, DB_PAO_ROW_MAPPER);
-    if (dbPao == null) {
+    List<DbPao> dbPao = tpsJdbcTemplate.query(sql, params, DB_PAO_ROW_MAPPER);
+    if (dbPao.isEmpty()) {
       throw new InternalTpsErrorException("Failed to get DbPao from object id");
     }
-    return dbPao;
+    return dbPao.get(0);
```

Comment:
nit: should we throw something if dbPao.size > 1? it might not be necessary if postgres is already enforcing uniqueness, though

Reply:
The lookup is on object_id which is the PK, so we don't need that check in this case.